Portfolio.html

```diff
@@ -1,225 +1,325 @@
+            <div class="title-box">
+              <h2 class="project-title">Tonic</h2>
+              <div class="canopy">
+                <div class="client">CANOPY</div>
+                <div class="counter"></div>
+                <div class="role">Back End Dev</div>
+                <div class="counter"></div>
+                <div class="year">2015</div>
+              </div>
+            </div>
+            <p class="description-box">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="tag-box">
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">Javascript</li>
+              </ul>
+            </div>
+            <div class="action-box">
+              <button class="btn btn-project">See Project</button>
+            </div>
+          </div>
         </div>
-        <ul class="tag01">
-          <li class="tag1">html</li>
-          <li class="tag2">css</li>
-          <li class="tag3">javascript</li>
-        </ul>
-
-        <button class="btn btn-project">See Project</button>
-      </div>
-      <!--END OF second CARD-->
-      <div class="singlecard">
-        <img class="portfoliopic" src="./images/Snapshoot-Portfolio3.png" alt="3rd-Snapshoot-Portfolio2" />
-        <div>
-          <h3 class="boldtext">Tonic</h3>
-          <a class="firsttxt">CANOPY</a>
-          <img class="smallimage" src="./images/Counter.png" alt="Counter" />
-          <a class="secondtxt">Back End Dev</a>
-          <img class="smallimage2" src="./images/Counter.png" alt="Counter" />
-          <a class="thirdtxt">2015</a>
-          <p class="longtext">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
+        <!-- End of first card -->
+        <div class="card-works">
+          <img
+            src="./images/work/multipost.png"
+            alt="Tonic Image"
+            class="snapshot-portfolio"
+          />
+          <div class="left-block">
+            <div class="title-box">
+              <h2 class="project-title">Multi-post Stories</h2>
+              <div class="canopy">
+                <div class="client">CANOPY</div>
+                <div class="counter"></div>
+                <div class="role">Back End Dev</div>
+                <div class="counter"></div>
+                <div class="year">2015</div>
+              </div>
+            </div>
+            <p class="description-box">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="tag-box">
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">Javascript</li>
+              </ul>
+            </div>
```

Comment:
- [ ] You've done well so far. Kindly check the broken images in your design and sort them out so your portfolio continues to look great as it is in other parts so far. Please check the image paths and fix them so all images load properly.

**See the broken images in your app**
![image](https://user-images.githubusercontent.com/50941074/171512434-18bec0c4-d136-4687-84aa-f749f4606d38.png)